### ISSUE TITLE
Easy links to versions of the decision graph?

### ISSUE THREAD

**Issue by timvisher** [clojure-liberator/liberator#226]
Is there a collection of links to versions of the decision graph. I noticed `initialize-context` show up and I don't have that in the version that I'm using.

**Comment by timvisher** [comment]
ping :)

**Comment by ordnungswidrig** [comment]
Pong :) I will release a fresh version on friday which contains "initialize-context".

**Comment by timvisher** [comment]
@ordnungswidrig Awesome. Actually what I'm asking for here is a versioned link to the decision graph. So [decision-graph 0.12.1](???) etc.

**Comment by ordnungswidrig** [comment]
Oh, yes,, simply tagging the graph with `since 1.x` would be hard because of changed edges. Actually liberator ships the graph in the jar and it's available in the "debug console" at `/x-liberator/requests` when you're using `wrap-trace`.

**Comment by timvisher** [comment]
I was unaware of that. Would it be very difficult to put it on the github.io site? Worse comes to worst I can just extract it from the jar. :)

**Comment by ordnungswidrig** [comment]
No, of course we can put it there. I need to think of a good way to link to old revisions. Because I'm sure in which revisions the graph actually changed and I only want to include those, I need to inspect all the recent jars. Doable but tedious. Help is appreciated.

**Comment by timvisher** [comment]
Ironically, I think only including the versions of the graph where they changed might be confusing. The use case that I'm envisioning is that I look at my project.clj and notice that I'm on `0.11` or `0.13` or whatever and I go to the 'decision graph' section of the github.io site and see links for all of the released versions, one of which is the one that I'm using (because I'm using a released version) and I click on that, not worrying about where my version lies in the midst of the versions where the decision graph actually changed.

It is true that it's not hard to do the mental arithmetic to decide that you want the last version before your version where the graph changed, it just seems to not be much of win given that you'd only be saving a few KBs on the server.

I can try to whip together a script that compares the versions though if we really want to go down that route.

**Comment by ordnungswidrig** [comment]
Good point! I think the best thing (maybe) is to provide a link for multiple versions link Graph for 0.9-0.11, for 0.12 and 0.13 e.g. (I made up the versions).

Having a script would be nice. I think this could be added to the gh-pages branch. Btw. this touches the same problem I see with api docs. Currently we don't link api doc in the documentation, because I did not figure a good way to call codox and integrate the output with the other docs.
